modules/cpmso-activating.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-activating_{context}"]
+= Activating the Control Plane Machine Set Operator
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.